install.sh

```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
+set -euo pipefail
+IFS=$'\n\t'
+
+
+function install-theme()
+{
+# Make temp directory
+cd "$(mktemp -d)" || exit 1
+
+# Install metro
+if [ -d ~/.steam ]
+then 
+read -p "Would you like to remove any previous installation of catppuccin? [Y/n] " -n 1 -r
+  echo 
+  if [[ $REPLY =~ ^[Yy]$ ]]
+  then
+      rm -fr ~/.steam/steam/skins/catppuccin
+  fi 
+  mkdir -p ~/.steam/steam/skins/
+  git clone https://github.com/minischetti/metro-for-steam ~/.steam/steam/skins/catppuccin
+  export install_path="$HOME/.steam/steam/skins/catppuccin"
+elif [ -d ~/.var/app/com.valvesoftware.Steam ]
+then 
+  echo "Would you like to remove any previous installation of catppuccin? [Y/n]"
+  echo
+  if [[ $REPLY =~ ^[Yy]$ ]]
+  then
+    rm -fr ~/.var/app/com.valvesoftware.Steam/.steam/steam/skins/catppuccin
+  fi 
+  mkdir -p ~/.var/app/com.valvesoftware.Steam/.steam/steam/skins
+  git clone https://github.com/minischetti/metro-for-steam ~/.var/app/com.valvesoftware.Steam/.steam/steam/skins/catppuccin
+  export install_path="$HOME/.var/app/com.valvesoftware.Steam/.steam/steam/skins/catppuccin"
+else
+  echo "Sorry, i couldn't find your steam installation"
+  echo "Please paste the path to '.steam' below"
+  echo "EXAMPLE: ~/.local/share/steam"
+  read -f -r "--> " steampath
+  mkdir "$steampath/steam/skins/catppuccin"
+  git clone https://github.com/minischetti/metro-for-steam "$steampath/steam/skins/catppuccin"
+  export install_path="$steampath/steam/skins/catppuccin"
+fi
+
+# Install the metro patch
+git clone https://github.com/redsigma/UPMetroSkin
+cp -r UPMetroSkin/"Unofficial 4.x Patch"/"Main Files [Install First]"/* "$install_path"
+
+# Finally install the catppuccin flavor of the users choosing
+git clone https://github.com/catppuccin/steam.git
+cd steam || exit 1
+flavors=(
+    'frappe'
+    'latte'
+    'mocha'
+    'macchiato'
+)
+PS3="Please select a flavor (TIP: You can input a number if you'd like): "
+select opt in "${flavors[@]}" ; do
+    [[ -n $opt ]] && break || {
```

Comment:
[Note that A && B || C is not if-then-else. C may run when A is true](https://www.shellcheck.net/wiki/SC2015)

break should be fine, but it would still be "best practice" to use if/else -- even to just avoid a shellcheck warning lol